src/slider/Slider.ts

```diff
@@ -126,7 +126,7 @@ export default class Slider extends SliderBase<SliderProperties> {
 		const outputNode = output ? output(value) : value + '';
 
 		const slider = v('div', {
-			classes: this.classes(css.inputWrapper),
+			classes: this.classes().fixed(css.inputWrapper),
```

Comment:
I've been wondering about situations like this, as I've run into them pretty frequently: should we always provide a themeable class on node's that have a fixed class? It seems odd that our components' themeable classes are only determined by what our default theme necessitates. What if some new theme needs to target `inputWrapper` to satisfy their mock ups, but can't simply because our default theme didn't need a themeable class there?

I don't think this should hold up the PR at all, and I think a pattern will become obvious from community usage and after our own initial theme is done.